### PR TITLE
donot limit action in opened & add column_name

### PR DESCRIPTION
**Pull request by SunRunAway**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +45,10 @@ jobs:
     steps:
     - name: Run assignment to one project
       uses: srggrs/assign-one-project-github-action@1.1.0
-      if: github.event.action == 'opened' # not required but speed up the action
+      if: github.event.action == 'labeled' && contains(github.event.issue.labels.*.name, 'test')
       with:
-        project: 'https://github.com/srggrs/assign-one-project-github-action/projects/2'
+        project: 'https://github.com/SunRunAway/assign-one-project-github-action/projects/1'
```
Comment (srggrs) [inline README.md]:
```suggestion
        project: 'https://github.com/srggrs/assign-one-project-github-action/projects/1'
```
please change others as above

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -9,8 +9,14 @@ jobs:
...
       uses: srggrs/assign-one-project-github-action@1.1.0
       if: github.event.action == 'opened'
       with:
         project: 'https://github.com/srggrs/assign-one-project-github-action/projects/2'
+    - name: Run assignment to one project 2
+      uses: srggrs/assign-one-project-github-action@1.1.0
+      if: github.event.action == 'labeled' && contains(github.event.issue.labels.*.name, 'bug')
+      with:
+        project: 'https://github.com/srggrs/assign-one-project-github-action/projects/3'
+        column_name: 'TODO'
```
Comment (SunRunAway) [inline .github/workflows/main.yml]: Hi, Thank you for your reply. Please create another project and name the default column as 'TODO' for example after merging.

File: README.md (hunk 1 above)
Comment (SunRunAway) [reply README.md]: done

**Comment by SunRunAway** [comment]
@srggrs Please help make a new release, Thanks.

**Comment by srggrs** [comment]
@SunRunAway I published the new action as a draft but I'm sure why my workflows fail. see https://github.com/srggrs/assign-one-project-github-action/actions/runs/52218347 and https://github.com/srggrs/assign-one-project-github-action/actions/runs/52205282

I get an error in parsing the workflow file....

**Comment by srggrs** [comment]
@SunRunAway fixed. The description field of the action was not a valid yaml

**Pull request merged**